EIPS/eip-20.md

```diff
@@ -167,27 +142,42 @@ MUST trigger on any successful call to `approve(address _spender, uint256 _value
 event Approval(address indexed _owner, address indexed _spender, uint256 _value)
 ```
 
+## Rationale
+
+### The approve and transferFrom flow
 
+`approve` and `transferFrom` can be used in conjunction with one another to allow smart contracts to take an action if and only if a transfer succeeds. This enables use cases such as decentralized exchanges, which can trustlessly swap one token for another.
 
-## Implementation
+### Decimals is optional
 
-There are already plenty of ERC20-compliant tokens deployed on the Ethereum network.
-Different implementations have been written by various teams that have different trade-offs: from gas saving to improved security.
+Some tokens might wish to use a base unit other than $10^{-18}$, and sometimes it doesn't make sense to have a fraction of a token. However, not every token might reasonably need to specify `decimals`. Thus, `decimals` is optional.
```

Comment:
I don't think we should invent content that wasn't there in the original proposal.

Reply:
It's a non-normative required section. If I were the person writing ERC-20 and decided to include this field, that would be why. If the original authors consent/agree with the motivation provided then I see nothing wrong with this.

Reply:
This is still more than is required to comply with current guidelines, and I don't think this qualifies as errata.